webhook_server_container/libs/github_api.py

```diff
@@ -1850,41 +1857,58 @@ def process_cherry_pick_command(self, issue_comment_id: int, command_args: str,
 
     def process_retest_command(self, issue_comment_id: int, command_args: str) -> None:
         _target_tests: List[str] = command_args.split()
-        for _test in _target_tests:
-            if _test == TOX_STR:
-                if not self.tox:
-                    msg: str = f"No {TOX_STR} configured for this repository"
-                    error_msg = f"{self.log_prefix} {msg}."
-                    self.logger.info(error_msg)
-                    self.pull_request.create_issue_comment(msg)
-                    return
+        _not_supported_retests: List[str] = []
+        _supported_retests: List[str] = []
+        _retests_to_func_map: Dict[str, Callable] = {
+            TOX_STR: self._run_tox,
+            PRE_COMMIT_STR: self._run_pre_commit,
+            BUILD_CONTAINER_STR: self._run_build_container,
+            PYTHON_MODULE_INSTALL_STR: self._run_install_python_module,
+        }
 
-                self.create_comment_reaction(issue_comment_id=issue_comment_id, reaction=REACTIONS.ok)
-                self._run_tox()
+        if not _target_tests:
+            msg = "No test defined to retest"
+            error_msg = f"{self.log_prefix} {msg}."
+            self.logger.debug(error_msg)
+            self.pull_request.create_issue_comment(msg)
+            return
 
-            elif _test == PRE_COMMIT_STR:
-                self.create_comment_reaction(issue_comment_id=issue_comment_id, reaction=REACTIONS.ok)
-                self._run_pre_commit()
+        if "all" in command_args:
```

Comment:
```suggestion
        if "retest all" in command_args:
```
?

Reply:
No, `process_retest_command` function get only the args, no `/retest` there.